noether_msgs/msg/PlaneSlicerRasterGeneratorConfig.msg

```diff
@@ -1,6 +1,18 @@
-float64 raster_spacing 	   # Offset between two rasters
-float64 point_spacing 	   # Required spacing between path points
-float64 raster_rot_offset  # Specifies the direction of the raster path in radians
-float64 min_segment_size   # The minimum segment size to allow when finding intersections; small segments will be discarded
-float64 min_hole_size 	   # A path may pass over holes smaller than this, but must be broken when larger holes are encounterd.
-float64 tool_offset 	   # How far off the surface the tool needs to be
+float64 raster_spacing 	    # Offset between two rasters
+float64 point_spacing 	     # Required spacing between path points
+float64 raster_rot_offset    # Specifies the direction of the raster path in radians. Rotates about short dimension of bounding box.
+float64 min_segment_size     # The minimum segment size to allow when finding intersections; small segments will be discarded
+float64 min_hole_size 	     # A path may pass over holes smaller than this, but must be broken when larger holes are encounterd.
+float64 tool_offset 	     # How far off the surface the tool needs to be
+bool generate_extra_rasters  # Whether an extra raster stroke should be added to the beginning and end of the raster pattern
```

Comment:
This perhaps could be made to be num_of_extra_rasters so that more than one could be added if needed.  Zero would be the default case

Reply:
This is a good idea. thanks